src/main/java/com/datasonnet/header/Header.java

```diff
@@ -68,7 +67,7 @@ public Header(String version,
                   boolean preserveOrder,
                   Map<String, Collection<MediaType>> namedInputs,
                   Map<String, MediaType> defaultInputs,
```

Comment:
We should probably remove `defaultInputs` here and `defaultOutput` below, since everything is fully specified with quality scores now

Reply:
So I deliberately left these on the constructor because it might useful for programatically defining defaults explicitly... but thinking about it it's probably best to just be consistent and use the quality params there also.